dc/core/resources/BorderRectangle.qml

```diff
@@ -39,7 +39,7 @@ Item {
         width: Style.borderWidth
         height: Style.borderWidth
         radius: Style.borderWidth
-        color: Style.inactiveBorderColor
+        color: "#00000000"
```

Comment:
The grabbed circle is still visible, others are invisible. I acctually don't know who is doing that magic...